CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 Preferably use **Added**, **Changed**, **Removed** and **Fixed** topics in each release or unreleased log for a better organization.
 
+## [1.0.8](https://github.com/quintoandar/hive-metastore-client/releases/tag/1.0.8)
+### Changed
+* Added if clause to check the default values for TableBuilder and SerdeInfoBuilder constructor
```

Comment:
```suggestion
* Added if clause to check the default values for `TableBuilder` and `SerdeInfoBuilder` constructor
```

Reply:
Actually, it is kind of tricky to have a **Changed** topic that starts with **Added** haha
What about:
```suggestion
* Replaced default values for `TableBuilder` and `SerdeInfoBuilder` constructors when no value is provided for specific lists and dicts
```

Reply:
yes, it's better hahaha